homeassistant/components/go2rtc/server.py

```diff
@@ -47,8 +64,17 @@ def __init__(self, hass: HomeAssistant, binary: str) -> None:
         self._binary = binary
         self._process: asyncio.subprocess.Process | None = None
         self._startup_complete = asyncio.Event()
+        self._stop_requested = False
+        self._watchdog_task: asyncio.Task | None = None
 
     async def start(self) -> None:
+        """Start the server."""
+        await self._start()
+        self._watchdog_task = asyncio.create_task(
```

Comment:
Why did you remove the hass background task?

Reply:
We call the `Server.stop` method when shutting down Home Assistant core, so I thought we can manage the task ourselves. Or would it be better if the task is both a background task and managed by us?

Reply:
Doesn't matter much. It may be convenient to not have to clean the task up manually.